classes/class.ilMumieTaskGradeListGUI.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+/**
+ * MumieTask plugin
+ *
+ * @copyright   2022 integral-learning GmbH (https://www.integral-learning.de/)
+ * @author      Vasilije Nedeljkovic(vasilije.nedeljkovic@integral-learning.de)
+ * @license     http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+/**
+ * This GUI provides a way to list grades and submission dates for a single user in a MUMIE task
+ */
+
+require_once('./Customizing/global/plugins/Services/Repository/RepositoryObject/MumieTask/classes/class.ilMumieTaskGradeSync.php');
+class ilMumieTaskGradeListGUI extends ilTable2GUI
+{
+    private $parent_gui;
+    private $postvar;
+    private $user_id;
+
+    public function __construct($parentObj)
+    {
+        parent::__construct($parentObj, 'displayGradeList');
+        $this->user_id = $_GET['user_id'];
+        $this->setId("user" . $_GET["ref_id"]);
+    }
+
+    public function init($parentObj)
```

Comment:
you don't need to pass `$parentObj` at this point. It's already been set by the constructor. You should be able access it by `$this>parent_obj`

Reply:
done